src/locales/en/common.json

```diff
@@ -2,7 +2,8 @@
   "languages": {
     "en": "English",
     "fr": "French",
-    "ar": "Arabic"
+    "ar": "Arabic",
+    "sw": "Swahili"
```

Comment:
This add is missing in fr/common.json and in ar/common.json

Reply:
i corrected it, it is now available